android-kotlin/app/src/main/java/com/mongodb/app/ItemsActivity.kt

```diff
@@ -0,0 +1,37 @@
+package com.mongodb.app
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import io.realm.Realm
+
+class ItemsActivity : AppCompatActivity() {
+    private lateinit var logoutButton: Button
+    private var user: io.realm.mongodb.User? = null
+    private var userRealm: Realm? = null
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_items)
+        user = myApp.currentUser()
+        if (user == null) {
+            startActivity(Intent(this, LoginActivity::class.java))
+        }
+        logoutButton = findViewById(R.id.button_log_out)
+        logoutButton.isEnabled = true
+        logoutButton.setOnClickListener{ (logout()) }
+    }
+
+    private fun logout() {
+        user?.logOutAsync {
```

Comment:
I'm not sure what the purpose of disabling and re-enabling is besides to avoid multiple calls that all generate errors if the user spams the logout button fast enough. Since this isn't likely, is the disabling and re-enabling unnecessary? or is it just good practice?